### DISCUSSION TITLE
AG & Logic Pro X Smart Controls?

### DISCUSSION THREAD

**Discussion by dsponaugle** [apohl79/audiogridder#321] [Q&A] [answered]
Does anyone know if it's possible to route LPX smart controls to plugin parameters contained within the AG plugin as if they were local on your machine?

**Comment by dsponaugle** [comment, marked as answer]
Got it sorted in the end. I had to manually add automation parameters for my plugin in AG and could then map to smart controls in LPX!

**Comment by CaBuffalo** [comment]
Thanks for sharing.

On Sun, Feb 7, 2021 at 8:45 PM dsponaugle <notifications@github.com> wrote:

> Got it sorted in the end. I had to manually add automation parameters for
> my plugin in AG and could then map to smart controls in LPX!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/321#discussioncomment-348215>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DHB5TN7NECXLRE34TS55T5PANCNFSM4WPJ3K7Q>
> .
>


-- 
Kevin Ma
Supported by Gmail.
